app/v1/models/errors/mtdErrors.scala

```diff
@@ -38,34 +44,41 @@ object TypeOfBusinessFormatError
 object AdjustedStatusFormatError
   extends MtdError("FORMAT_ADJUSTED_STATUS", "The supplied adjusted status format is invalid")
 
+object FormatAdjustmentValueError extends MtdError("FORMAT_ADJUSTMENT_VALUE", "") {
+  def withFieldName(fieldName: String): MtdError = this.copy(message = s"The format of the '$fieldName' value is invalid")
+}
+
 object SelfEmploymentIdFormatError extends MtdError("FORMAT_SELF_EMPLOYMENT_ID","The supplied self-employment ID format is invalid")
 
+object BsasIdFormatError extends  MtdError("FORMAT_BSAS_ID", "The provided Business Source Adjustable Summary ID format is invalid")
```

Comment:
Message should be "The format of the BSAS ID is invalid"

Reply:
I had changed this but it probably snuck back in on the rebase